client/cypress/support/commands.js

```diff
@@ -6,7 +6,106 @@ import "@testing-library/cypress/add-commands";
 
 const { each } = Cypress._;
 
-Cypress.Commands.add("login", (email = "admin@redash.io", password = "password") => {
+const LOGIN_NAME = process.env.CYPRESS_LOGIN_NAME || "Example Admin";
+const LOGIN_EMAIL = process.env.CYPRESS_LOGIN_EMAIL || "admin@redash.io";
+const LOGIN_PASSWORD = process.env.CYPRESS_LOGIN_PASSWORD || "password";
+const ORG_NAME = process.env.CYPRESS_ORG_NAME || "Redash";
+
+Cypress.Commands.add("setup", () => {
```

Comment:
How can I call this from terminal?
I got stuck so it never pass login screen on any test after I typed `npm run cypress all`

Reply:
this cypress command can be called using `cy.setup`, it's added to client/cypress/support/commands.js
it's executed before e2e tests execution. you need to run `yarn cypress run-ci`

Reply:
adding .env file is a responsibility of a developer. run "make .env"